src/Phinx/Config/ConfigInterface.php

```diff
@@ -81,6 +81,13 @@ public function getDefaultEnvironment();
      */
     public function getAlias($alias);
 
+    /**
+     * Get all the aliased values.
+     *
+     * @return array|null
```

Comment:
Why not just always array? would be cleaner IMO.

Reply:
Fair enough.